### DISCUSSION TITLE
Another Matrix screensaver

### DISCUSSION THREAD

**Discussion by toncho11** [jbruchon/elks#1422] [General]
I have retrieved the source code of a Matrix screensaver.

https://github.com/toncho11/16bit/tree/main/fallscr

@ghaerr I wonder if it is the same or better than the one currently in ELKS. 
And if this one can be ported to ELKS for fun :)

**Comment by ghaerr** [comment]
Hi @toncho11,

Thanks for the link for another Matrix. That one is written in ASM, and uses hardware to scroll it on the screen, so its not very portable. Our current Matrix also runs on connected terminals, and translates IBM PC codepage 437 to unicode for ANSI terminals (for those who care!) :)

Nonetheless, one matrix is enough for me to port, but others are welcome to take a stab at it!
